be/src/storage/persistent_index.cpp

```diff
@@ -662,7 +662,7 @@ class FixedMutableIndex : public MutableIndex {
         return Status::OK();
     }
 
-    size_t dump_bound() { return _map.dump_bound(); }
+    size_t dump_bound() { return _map.empty() ? sizeof(size_t) : _map.dump_bound(); }
```

Comment:
Please add some comment in the code to explain why return `sizeof(size_t)` when _map is empty

Reply:
done, added in https://github.com/StarRocks/starrocks/pull/7450/commits/acfebf97903645f6d6f6a23f78c07893f8efca31